src/serialization/SerializationService.ts

```diff
@@ -47,10 +48,17 @@ export class SerializationServiceV1 implements SerializationService {
         this.registerGlobalSerializer(serializationConfig.globalSerializer);
     }
 
-    toData(object: any, partitioningStrategy: any = this.defaultPartitionStrategy): Data {
+    toData(object: any | PartitionAware<any>, partitioningStrategy: any = this.defaultPartitionStrategy): Data {
```

Comment:
I think `any` here already has a wider "scope" than `PartitionAware`, so it can be removed.